### ISSUE TITLE
Share work information with social media accessibility test 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3987]
### Descriptive summary

Descriptive summary
QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

### Video of Tab Test  

https://cl.ly/0c4743539965/download/Screen%252520Recording%2525202019-09-12%252520at%25252008.27%252520PM.mov

At https://nurax-dev.curationexperts.com/concern/images/tq57nr202?locale=en I am unable to access the social media links along with other links on FireFox and Safari. 

Chrome working well.

**Comment by no-reply** [comment]
i wasn't able to reproduce this in `Mozilla Firefox 86.0`:

 Hyrax Version: 3.0.0.pre.rc4
Hyrax Branch: master
Hyrax Revision: 3579d7a5b5d8d04562cdec8c275f82f1e9b7f1e8
© 2021 Samvera-Hyrax
Nurax Version
ENV: production
SHA: 816ad1513498e7ad0b98f5092e94074550c28789
Branch: 2021-03-17-10-00
Updated: 17 Mar 2021 15:00:26

**Comment by jlhardes** [comment]
I wasn't able to reproduce this on Safari 17.3 but I can see that tab order is skipping over social media links and some other links in Firefox 122.0. Chrome is still working well.
